src/createStore.js

```diff
@@ -71,6 +71,10 @@ export default function createStore(reducer, initialState, enhancer) {
    * @returns {any} The current state tree of your application.
    */
   function getState() {
+    if (isDispatching) {
+      throw new Error('Reducers may not access store state.')
```

Comment:
Yeah, I was just going with similar format to ```dispatch``` message. I like the more descriptive messages though. How about for ```subscribe``` and ```unsubscribe```?